src/main/java/org/jabref/gui/actions/StandardActions.java

```diff
@@ -70,7 +70,7 @@ public enum StandardActions implements Action {
     EXPORT_SELECTED(Localization.lang("Export selected entries"), KeyBinding.EXPORT_SELECTED),
     CONNECT_TO_SHARED_DB(Localization.lang("Connect to shared database"), IconTheme.JabRefIcons.CONNECT_DB),
     PULL_CHANGES_FROM_SHARED_DB(Localization.lang("Pull changes from shared database"), KeyBinding.PULL_CHANGES_FROM_SHARED_DATABASE),
-    CLOSE_LIBRARY(Localization.lang("Close"), Localization.lang("Close the current library"), IconTheme.JabRefIcons.CLOSE, KeyBinding.CLOSE_DATABASE),
+    CLOSE_LIBRARY(Localization.lang("Close library"), Localization.lang("Close the current library"), IconTheme.JabRefIcons.CLOSE, KeyBinding.CLOSE_DATABASE),
```

Comment:
you will need to fix this in en.properties as well.  file https://devdocs.jabref.org/code-howtos/localization.html

Reply:
There is this one: 
<img width="376" alt="image" src="https://github.com/JabRef/jabref/assets/127773292/5602678d-1218-40ba-b618-a6a15aacc918">

So I just delete this one:
<img width="512" alt="image" src="https://github.com/JabRef/jabref/assets/127773292/43c880bd-7407-4e58-8f80-c72f1badaf15">
?

Reply:
Add a new one with close library. I guess Close is used in other parts of the GUI as well
Just run LocalizationConsistencyTest it will tell you